src/managers/EventManager.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import type { EventPublisher } from 'spark-protocol';
+import type { DeviceAttributeRepository, Event, EventData } from '../types';
+
+class EventManager {
+  _eventPublisher: EventPublisher;
+  _deviceAttributeRepository: DeviceAttributeRepository;
+
+  constructor(
+    deviceAttributeRepository: DeviceAttributeRepository,
+    eventPublisher: EventPublisher,
+  ) {
+    this._deviceAttributeRepository = deviceAttributeRepository;
+    this._eventPublisher = eventPublisher;
+  }
+
+  _filterEvents = (
```

Comment:
I'm not sure about this function. maybe I'll have to move filtering in EventPublisher and add userID to Event, because
it could be needed for device <-> device events subscriptions. haven't figured out it yet.